frontend/src/static/js/components/webstatus-columns-dialog.ts

```diff
@@ -58,41 +68,70 @@ export class WebstatusColumnsDialog extends LitElement {
 
   handleSave() {
     const newColumns: string[] = [];
+    const columnOptions: string[] = [];
     this.shadowRoot!.querySelectorAll('sl-checkbox').forEach(cb => {
+      if (cb.classList.contains('column-option') && !cb.checked) {
+        columnOptions.push(cb.value);
+      }
       if (cb.checked) {
         newColumns.push(cb.value);
       }
     });
```

Comment:
After adding the column class in my other suggestion, you can have two querySelectorAll methods. One for the columns and one for the column option.
```suggestion
    this.shadowRoot!.querySelectorAll<SlCheckbox>('sl-checkbox.column').forEach(
      cb => {
        if (cb.checked) {
          newColumns.push(cb.value);
        }
      }
    );
    this.shadowRoot!.querySelectorAll<SlCheckbox>(
      'sl-checkbox.column-option'
    ).forEach(cb => {
      if (cb.checked) {
        columnOptions.push(cb.value);
      }
    });
```